### DISCUSSION TITLE
How can I get a different color schemes?

### DISCUSSION THREAD

**Discussion by LAPIII** [WinMerge/winmerge#1724] [Q&A]
Can I download,, create, or modify color schemes?

**Comment by sdottaka** [comment]
WinMerge's color scheme definition files are stored in `C:\Program Files\WinMerge\ColorSchemes`.
You can create new color schemes by editing them or copying and renaming them.

**Reply by Kissaki** [reply]
I copied a theme file but in the settings it doesn't show up

**Comment by MartinCowen** [comment]
I've just written a Windows C# program to solve this problem for those of us who don't quite like the current set of color schemes available. Download the exe from the Release folder or build yourself from source using Visual Studio. https://github.com/MartinCowen/ThemeWinMerge

**Reply by Kissaki** [reply]
can open Modern.ini but throws a format exception when trying to open midnight or solarized dark.
